tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "resolveJsonModule": true,
     "experimentalDecorators": true,
     "strictNullChecks": true,
-    "types": ["jest"],
+    "types": ["jest", "@testing-library/jest-dom"],
```

Comment:
The problem with this is that it makes these types available everywhere, but we only want them available in test code. It should be an error to reference these globals/types in non-test code. The conventional way to handle this for `@testing-library/jest-dom` is to import it in the `setupTests.ts` file.

Reply:
I don't know exactly why it behaves the way it does, but I was able to fix it by:
1. Renaming `setupTests.js` to `setupTests.ts`
2. Making the same change in the jest configuration in `package.json`
3. Using `import `@testing-library/jest-dom` instead of `require`ing it

Reply:
@ksweson Thanks! I can confirm that fixes the problem.